poetry.lock

```diff
@@ -768,7 +768,7 @@ name = "pycparser"
 version = "2.21"
 description = "C parser in Python"
 optional = false
-python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+python-versions = "*"
```

Comment:
It's unclear why this changed. I'm using poetry 1.6.1 (latest).

Reply:
I don't suppose that's a big issue. I would just be interested why it happened. If `pycparser` changed its Python version requirements then I'd assume that it'd bump its version somehow too.